versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application-on-an-intra-cluster-slice.mdx

```diff
@@ -1,32 +1,40 @@
 
 ## Introduction
-Bookinfo ([Bookinfo Application](https://istio.io/latest/docs/examples/bookinfo/)) is a sample application from Istio composed of four separate microservices productpage, details, reviews, and ratings. This topic describes the steps to install the Bookinfo application on a single cluster.
+Bookinfo ([Bookinfo Application](https://istio.io/latest/docs/examples/bookinfo/)) is a sample application from Istio 
+composed of four separate microservices productpage, details, reviews, and ratings. 
+This topic describes the steps to install the Bookinfo application on a single cluster.
 
  
 
 ## Prerequisites
 Before you begin, ensure the following prerequisites are met:
 
-- You have the KubeSlice Controller components and worker cluster components on the same cluster. For more information, see [Installing KubeSlice](/versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/installing-kubeslice.mdx).
+- You have the KubeSlice Controller components and worker cluster components on the same cluster. For more information, 
+see [Installing KubeSlice](/versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/installing-kubeslice.mdx).
 - Before creating the slice, create a `bookinfo` namespace in all participating worker.
   Use the following command to create the `bookinfo` namespace:
   ```
   kubectl create ns Bookinfo
   ```
 
 ## Creating the Slice
-To install the BookInfo Application on a single cluster, create a slice without Istio. For more information, see [Creating a Slice](/versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/creating-a-slice.mdx).
+To install the BookInfo Application on a single cluster, create a slice without Istio. For more information, 
+see [Creating a Slice](/versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/creating-a-slice.mdx).
 
 ## Creating the Bookinfo Deployment YAML Files
-Using the templates below, create the necessary .yaml files to deploy the Bookinfo application. All fields in the template will remain the same except for the <slice name\> which must be replaced with the name of your slice.
+Using the templates below, create the necessary .yaml files to deploy the Bookinfo application. All fields in the 
+template will remain the same except for the <slice name\> which must be replaced with the name of your slice.
 
-:::note
-These instructions will guide you through deploying the Product Page service to a cluster we will refer to as the <productpage cluster\>, and the remaining services as well as service exports will be deployed to a cluster referred to here as the <services cluster\>.
+:::info
+These instructions will guide you through deploying the Product Page service to a cluster we will refer to as 
+the <productpage cluster\>, and the remaining services as well as service exports will be deployed to a 
```

Comment:
instead of adding escaping character before greater than size;  > in <productpage cluster\>

try adding backticks `<productpage clusters>`

Reply:
Updated the <place holders> with backticks in all the 4 tutorials doc.

Reply:
Also, update the numbered list for the steps.